tests/sources/test_spot_price_manual_source.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from telliot_feeds.sources.manual_sources.spot_price_input_source import SpotPriceManualSource
+from telliot_feeds.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_manual_spot_price(monkeypatch):
```

Comment:
Add test for bad input using `unittest.mock.patch` & and a list of inputs as the `side_effect`. See the test for manual snapshot feed as an example.